File: 2645-pass-the-pillow/pass-the-pillow.py
class Solution(object):
    def passThePillow(self, n, time):
        """
        :type n: int
        :type time: int
        :rtype: int
        """
        if (time/(n-1))%2:
            return n-(time%(n-1))
        else:
            return (time%(n-1))+1

File: 2645-pass-the-pillow/pass-the-pillow.cpp
class Solution {
public:
    int passThePillow(int n, int time) {
        if(time<n){
            return time+1;
        }else{
            int rem=time%(n-1);
            if(((time/(n-1))%2)){
                return n-rem;
            }else{
                return rem+1;
            }
        }
    }
};